### PR TITLE
[tflite_export] add Error Message, Fix Application @open sesame 10/05 15:54

### PR DESCRIPTION
**Pull request by DonghakPark**
Now, an Error occurs when export resnet to tflite format

Previously, the prop of the layer was changed in the process of comparing the values with the Pytorch. But, Tflite only supports ```same``` and ```valid``` padding, but the current application use value as 1,1.

To correct this, a macro was added, and a more detailed error message was added to the tflite exporter.

### CODE REVIEW

**Review by SeoHyungjun — APPROVED** [review]
LGTM!

**Review by baek2sm — APPROVED** [review]
LGTM.

But we need to check later whether implementation of ResNet is right because "stride 2 and padding same" is a little ambiguous.

**Comment by DonghakPark** [comment]
> LGTM.
> 
> But we need to check whether implementation of ResNet is right later because "stride 2 and padding same" is a little ambiguous.

Yes, that right! But in tensorflow they only use "same", "valid" padding, so for export as tflite format we need this part.

File: nntrainer/utils/node_exporter.cpp [hunk 1]
```diff
@@ -197,7 +195,10 @@ void Exporter::saveTflResult(
   auto strides = std::get<std::array<props::Stride, CONV2D_DIM>>(props);
   assert(strides.size() == CONV2D_DIM);
   auto padding = std::get<props::Padding2D>(props).get();
-  assert(padding == "same" || padding == "valid");
+  if (padding != "same" && padding != "valid") {
+    throw std::runtime_error("Unsupported padding type");
```
Comment (myungjoo) [inline nntrainer/utils/node_exporter.cpp:199]: Give more info for the error. E.g., 
```
        std::ostringstream ss;
        ss << "Unsupported padding type; \"" << padding << "\" is not supported. Use \"same\" or \"valid\".";
        throw std::runtime_error(ss.str());
    }
```

Comment (myungjoo) [reply nntrainer/utils/node_exporter.cpp:199]: If you have thrown an error, do you still need to do assert?

Comment (DonghakPark) [reply nntrainer/utils/node_exporter.cpp:199]: Thank you for Review, Fix!!

**Review by myungjoo — APPROVED** [review]
